src/docs/development/ios-14.md

```diff
@@ -46,6 +48,67 @@ This crash _doesn't_ happen when using the iOS simulator.
 For more information,
 see [Issue 62531]({{site.github}}/flutter/flutter/issues/62531).
 
+## Launching Flutter with Flutter tools
+
+Due to low-level changes in iOS’s debugger mechanisms,
+developers using versions of Flutter earlier than 1.22 beta
+won't be able to launch apps (by using `flutter run`
+or a Flutter IDE plugin) on physical iOS devices
+running iOS 14. This affects **debug**, **profile**, and
+**release** builds. Simulator builds, add-to-app modules,
+and running directly from Xcode are unaffected. 
+
+Developers who have upgraded to Flutter 1.22 beta can build,
+test, and deploy to iOS without issue.
```

Comment:
Add `Upgrading to 1.20.4 stable can also build and deploy on iOS 14 but not debug. ` since we cherry-picked.